server/controllers/userController.js

```diff
@@ -211,6 +213,36 @@ class UserController {
     );
     return util.successStatus(res, 200, 'Password reset successfully');
   }
+  static async assignUserRole(req, res) {
+    const { id } = req.user ? req.user : req.query;
+    const { email, role } = req.body;
+    if( !id ) { return util.errorStatus(res, 401, 'Not Authorized'); }
+    try {
+        const superAdmin = await models.Users.findByPk(id);
+        if(!superAdmin || superAdmin.role !== 'super_admin') { return util.errorStatus(res, 401, 'Not Authorized'); }
+        const user = await models.Users.findOne({where: {email}});
+        if(!user) { return util.errorStatus(res, 404, 'User does not exists'); }
+        models.Users.update({ role }, {where: { email } });
+        return util.successStatus(res, 200, 'Role Assigned successfully', {
+          assignedBy: superAdmin.firstName,
+          id: user.id,
+          firstName: user.firstName,
+          lastName: user.lastName,
+          email:  user.email
+        })
+    } catch(err) {
+        return util.errorStatus(res, 500, err.message);
+    }
+  }
+
+  // static async checkUserRole(req, res, next) {
```

Comment:
Remove unused comments

Reply:
Please format your code well and also add a few line spaces to make them readble.